packages/inter-protocol/src/psm/psm.js

```diff
@@ -196,33 +197,52 @@ export const start = async (zcf, privateArgs, baggage) => {
     totalStableProvided = AmountMath.add(totalStableProvided, asStable);
   };
 
-  /**
-   * @param {ZCFSeat} seat
-   */
-  const swapHook = seat => {
-    assertProposalShape(seat, {
-      give: { In: null },
-    });
+  /** @param {ZCFSeat} seat */
+  const giveStableHook = seat => {
```

Comment:
these changes are in master now. please rebase for a smaller review.

Reply:
done